### ISSUE TITLE
Hero and Nav

### ISSUE THREAD

**Issue by VanessaVenti** [harvard-lts/CURIOSity#40]
- Vertically center menu items and move the search box over the hero image
- Color contrast insufficient on keyboard nav, esp. on Curated Features. 
- Secondary nav should be updated to match HL design
- Consider reducing size of hero image or condensing altogether when not on the homepage. Users have given feed back that it takes up too much space on smaller screens when viewing search results and item detail pages.
- The teal could be removed and go back to the default styling. There is a known bug where the teal and search box doesn't always load (see screenshot below), refreshing the page fixes the issue. The bug is sporadic and therefore difficult to replicate. Phil has investigated this bug and may have more info.

<img width="1434" alt="Partially loaded theme bug" src="https://user-images.githubusercontent.com/60190174/165402179-3f6e6671-9c10-447f-ad96-6b0417da570d.png">

**Comment by summer-cook** [comment]
- Vertically center menu items and move the search box over the hero image
_Where would you like the search bar to be placed on the hero image?_

- Color contrast insufficient on keyboard nav, esp. on Curated Features.
_Where exactly is the color contrast insufficient? Are you talking about on the curated features page, or in the nav bar?_

- Secondary nav should be updated to match HL design
_Do you have a screenshot of this one? the last point on here seems to also address the secondary nav, so just want to clarify what i should do there_

- Consider reducing size of hero image or condensing altogether when not on the homepage. Users have given feed back that it takes up too much space on smaller screens when viewing search results and item detail pages.
_this is definitely doable_

-The teal could be removed and go back to the default styling. There is a known bug where the teal and search box doesn't always load (see screenshot below), refreshing the page fixes the issue. The bug is sporadic and therefore difficult to replicate. Phil has investigated this bug and may have more info.
_is this talking about the teal in the secondary nav bar? the default is that transparent black with white writing. So if we go back to the default thats what it would be. But I am assuming you also wanted all the links centered/spaced evenly across the screen like in these: https://www.sketch.com/s/83a22487-2bee-4f2e-a416-18116150d32a/a/oY49w8D/play_

**Comment by VanessaVenti** [comment]
@summer-cook - no problem at all, sorry for not having flushed out mock-ups on this! We will get those to you tomorrow and would love to meet with you to review too. In the meantime, here's what we're thinking.

You can use this mock up as a guide for now, but stay tuned for an updated mock up:
![Screen Shot 2022-04-28 at 12 08 54 PM](https://user-images.githubusercontent.com/60190174/165797134-5dfb1c40-6976-4571-8e9e-cc97639bdc4f.png)

- The one exception is we would like to keep the notched in sides (like on the live site) where the teal nav doesn't quite extend to the edges. The search box would then be notched in above it as well so the right edges would be aligned. 
- We also know the sizing/placement of the title will need some adjusting too as we have some long titles and this design doesn't account for when there is a subtitle. See these site as examples:
  -  https://curiosity.lib.harvard.edu/slavery-abolition-emancipation-and-freedom
  - https://curiosity.lib.harvard.edu/expeditions-and-discoveries
  - https://curiosity.lib.harvard.edu/archived-harvard-university-online-course-catalogs-calendars (no subtitle, but very long title)

- Color contrast insufficient on keyboard nav, esp. on Curated Features.
_Where exactly is the color contrast insufficient? Are you talking about on the curated features page, or in the nav bar?_
VV: Here is a screenshot of the issue where the Browse link is highlighted with insufficient contrast. This happens using the keyboard to navigate the site. We will create a mock up with a solve.
![Screen Shot 2022-04-28 at 12 20 30 PM](https://user-images.githubusercontent.com/60190174/165799559-e509b305-cd25-4773-9091-1affc09ce643.png)
 
- Secondary nav should be updated to match HL design
_Do you have a screenshot of this one? the last point on here seems to also address the secondary nav, so just want to clarify what i should do there_
VV: we will create a mock up for you

- Consider reducing size of hero image or condensing altogether when not on the homepage. Users have given feed back _that it takes up too much space on smaller screens when viewing search results and item detail pages.
this is definitely doable_
VV: The new mock ups will reduce the size a bit. Since we decided to move the search box over the hero image we'll have to think more about what a condensed version might look like when not on the homepage. We can leave it the same on all pages for now.

- The teal could be removed and go back to the default styling. There is a known bug where the teal and search box doesn't always load (see screenshot below), refreshing the page fixes the issue. The bug is sporadic and therefore difficult to replicate. Phil has investigated this bug and may have more info.
_is this talking about the teal in the secondary nav bar? the default is that transparent black with white writing. So if we go back to the default thats what it would be. But I am assuming you also wanted all the links centered/spaced evenly across the screen like in these: https://www.sketch.com/s/83a22487-2bee-4f2e-a416-18116150d32a/a/oY49w8D/play_
VV: We're thinking we'll stick with the teal for now, but Enrique and I are meeting tomorrow AM to create the mock ups where we'll make final decisions. Yes, please do center and evenly space the nav items like in the screenshot.

**Comment by summer-cook** [comment]
Its possible we want to close this one, since many of these decisions were changed based on Enrique's mockups?

**Comment by VanessaVenti** [comment]
Agreed, this one is now out of date based on the new designs. I'll go ahead and close it.
